charts/galoy/templates/admin-panel-deployment.yaml

```diff
@@ -0,0 +1,30 @@
+{{ if (index .Values "admin-panel").enabled }}
```

Comment:
what is `index` for here?

Reply:
`{{if .Values.admin-panel.enabled}}` is invalid helm syntax due to the hyphen

Reply:
Shouldn't we be calling it `adminPanel`? I think `camelCase` is idiomatic for helm charts right?